### DISCUSSION TITLE
How do I get this working with Rainmeter?

### DISCUSSION THREAD

**Discussion by YISTECH** [ciderapp/Apple-Music-Electron#190] [Q&A]
I use rainmeter on a daily basis with iTunes. How do I get this to work with Rainmeter?

**Comment by quacksire** [comment]
What's rainmeter?

**Reply by YISTECH** [reply]
it’s an app on windows which you can use for widgets. Quite popular. Honestly a bit suprised you haven’t heard of it but yeah, that’s what it is
